front/app/api/ideas/useUpdateIdea.ts

```diff
@@ -48,7 +48,7 @@ const useUpdateIdea = () => {
       }
 
       streams.fetchAllWith({
-        apiEndpoint: [`${API_PATH}/analytics`, `${API_PATH}/topics`],
+        apiEndpoint: [`${API_PATH}/analytics`],
```

Comment:
Do we need to invalidate topics using `invalidateQueries` instead? Or was it unnecessary to invalidate topics in this case?

Reply:
I don't think the general topics api should be invalidated when an idea is updated. I think it might have been a mistake adding this here.